src/components/Progress/ASABProgress.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Progress } from 'reactstrap';
+
+import './ASABProgress.scss';
+
+export function ASABProgress({
+	showLabel = true,
+	children,
+	...props
+}) {
+	// Handle both numeric and string values;
+	let progressValue;
+	let isNumeric = typeof props.value === 'number';
+
+	// Return null if value is not a number or string
+	if (!isNumeric && typeof props.value !== 'string') {
+		return null;
+	}
+
+	if (isNumeric) {
+		// Ensure value is a number
+		progressValue = Number(props.value);
+		if (!Number.isFinite(progressValue)) {
+			progressValue = 0;
+		}
+		// Clamp between 0-100 & round to nearest integer
+		progressValue = Math.round(Math.min(Math.max(progressValue, 0), 100));
+	} else {
+		// For strings, show indeterminate progress
+		progressValue = 100;
```

Comment:
I dont think this is intended. Sorry, I might have mislead you by my previous commentaries. The value always have to be present to display the colored part of the progress, but the text may differ (it does not need to always display the percentage, but the text like `High`, `Low`, etc..

Please have a look here for inspiration:

https://reactstrap.github.io/?path=/docs/components-progress--progress